src/test/debugger/android/androidPlatform.test.ts

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as Q from "q";
+import * as path from "path";
+import * as mockFs from "mock-fs";
+
+import {AndroidPlatform} from "../../../debugger/android/androidPlatform";
+import {IRunOptions} from "../../../common/launchArgs";
+import {FileSystem} from "../../../common/node/fileSystem";
+import {ReactNative022} from "../../../test/resources/reactNative022";
+import {DeviceHelper} from "../../../test/resources/simulators/deviceHelper";
+import {AVDManager} from "../../../test/resources/simulators/avdManager";
+import {FakeExtensionMessageSender} from "../../../test/resources/fakeExtensionMessageSender";
+import {ExtensionMessage} from "../../../common/extensionMessaging";
+import {RecordingsHelper} from "../../resources/recordingsHelper";
+import {RemoteExtension} from "../../../common/remoteExtension";
+
+import "should";
+
+// TODO: Launch the extension server and test the logcat functionality
+
+suite("androidPlatform", function () {
+    this.timeout(5000); // Currently some tests are taking about 2 seconds to finish
+    suite("debuggerContext", function () {
+        const projectRoot = "C:/projects/SampleApplication_21/";
+        const androidProjectPath = path.join(projectRoot, "android");
+        const applicationName = "SampleApplication";
+        const androidPackageName = "com.sampleapplication";
+        const genericRunOptions: IRunOptions = { projectRoot: projectRoot };
+
+        let fileSystem: FileSystem;
+        let deviceHelper: DeviceHelper;
+        let simulatedAVDManager: AVDManager;
+        let reactNative: ReactNative022;
+        let fakeExtensionMessageSender: FakeExtensionMessageSender;
+        let androidPlatform: AndroidPlatform;
+
+        setup(() => {
+            // Configure all the dependencies we'll use in our tests
+            fileSystem = new FileSystem({ fs: mockFs.fs({}) });
+            deviceHelper = new DeviceHelper(fileSystem);
+            simulatedAVDManager = new AVDManager(deviceHelper);
+            reactNative = new ReactNative022(deviceHelper, fileSystem);
+            fakeExtensionMessageSender = new FakeExtensionMessageSender();
+            androidPlatform = new AndroidPlatform(genericRunOptions, {
```

Comment:
Yes, I missed that.
Changed for:
            androidPlatform = createAndroidPlatform(genericRunOptions);
And move functions before the setup();